composer.json

```diff
@@ -55,7 +55,11 @@
         "php-cs-fixer": "bin/php-cs-fixer fix -v --ansi",
         "phpspec": "bin/phpspec run --format=pretty --ansi",
         "phpspec-coverage": "bin/phpspec run -f progress -c phpspec-coverage.yml --ansi",
-        "behat": "bin/behat"
+        "behat": "bin/behat",
+        "docker:build:php74": "docker build -t pugx/poser:php74 -f .circleci/images/php74/Dockerfile .",
```

Comment:
why docker build here?

Reply:
for documentation, do you see any problems?

Reply:
No problems, only to understand the meaning of those lines inside composer.json 👍

Reply:
it's a practice I've often seen make in node projects in package.json and I think I'll try it! :)